README.md

```diff
@@ -1,7 +1,26 @@
-# chat_c
-## How to install:
-Extract ZIP file and place in mods/. Alternatively, you can simply "git clone https://github.com/wilkgr76/chat_c" from within the mods directory.
-## How to use:
+# Coloured Chat
+Allows players to colour their chat messages.
+(See the example below.)
+
+## Installation
+- Unzip the archive, rename the folder to "soccer" (**without the quotes**) and
```

Comment:
should be chat_c

Reply:
This is outdated, the text says `chat_c`.

Reply:
yes I resolved it already.